docs/security-encryption.md

```diff
@@ -69,7 +69,7 @@ To enable end-to-end encryption in Puslar, complete the following steps.
 
      ```shell
      openssl genrsa -out test_rsa_privkey.pem 2048
-     openssl rsa -in test_rsa_privkey.pem -pubout -outform pkcs8 -out test_rsa_pubkey.pem
+     openssl rsa -in test_rsa_privkey.pem -outform PEM -pubout -out test_rsa_pubkey.pem
```

Comment:
I wonder if there's a need to also convert it to PKCS8 format as it has been before?

If it is needed, it could be achieved by adding one more step:
```
openssl pkcs8 -topk8 -inform PEM -outform PEM -nocrypt -in test_rsa_pubkey.pem -out test_rsa_pubkey.pk8
```

Perhaps some library requires it in PKCS8 format?

Reply:
Perhaps it's already the default?

Reply:
I wonder why `-outform pkcs8` was used in the first place.